spock-core/src/main/java/org/spockframework/compiler/AbstractDeepBlockRewriter.java

```diff
@@ -149,6 +149,12 @@ public final void visitClosureExpression(ClosureExpression expr) {
       currSpecialMethodCall = NoSpecialMethodCall.INSTANCE; // unrelated closure terminates currSpecialMethodCall scope
     }
     try {
+      Statement code = expr.getCode();
+      if (!(code instanceof BlockStatement)) {
+        BlockStatement block = new BlockStatement();
+        block.addStatement(code);
+        expr.setCode(block);
```

Comment:
Strange that the code block is marked as not covered, is there a test case missing?

Reply:
If I'm not mistaken it should already do that. https://github.com/spockframework/spock/blob/f214a8a20e7a0bc13d8da632ac51d24fae58211e/spock-specs/specs.gradle#L96-L101